test/api/TestFactory.js

```diff
@@ -30,23 +32,10 @@ window.testConfig = {
 };
 
 /**
- * @param {function} [logger_level] - A function returning the logger level.
  * @returns {Window.TestFactory} A TestFactory instance.
  * @constructor
  */
-window.TestFactory = function(logger_level) {
-  if (!logger_level) {
-    logger_level = z.util.Logger.prototype.levels.OFF;
-  }
-
-  const initialLoggerOptions = z.config.LOGGER.OPTIONS;
-  Object.keys(initialLoggerOptions.domains).forEach(domain => {
-    initialLoggerOptions.domains[domain] = logger_level;
-  });
-  initialLoggerOptions.level = logger_level;
-
-  return this;
-};
+window.TestFactory = function() {};
```

Comment:
Can we remove that?

Reply:
unfortunalty no, we still do `new TestFactory` everywhere. 
But in the long term, yes we will remove that class entirely :)